src/esp/assets/ResourceManager.cpp

```diff
@@ -1428,7 +1428,7 @@ scene::SceneNode* ResourceManager::createSemanticRenderAssetInstance(
     scene::SceneNode* parent,
     DrawableGroup* drawables) {
   if (creation.isTextureBasedSemantic()) {
-    // Treat texture-based semantic meshes as General/Primitves.
+    // Treat texture-based semantic meshes as General/Primitives.
```

Comment:
Nit: I would argue the grammar was better before. General/Primitive is a category.